docker/docker-compose.test.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.4"
+
+services:
+  unit-test:
+    build:
+      args:
+        NODE_ENV: development
+        PORT: ${PORT}
+      context: ../
+      target: development
+    command: npm run test
+    volumes:
+      - "../server/:/home/node/server/"
+      - "../test/:/home/node/test/"
```

Comment:
are the mounts for server and tests necessary? The test only writes to the test-output folder.

It's worth noting that a limitation of WSL with Docker is that it doesn't like relative paths, or the /mnt folder.
To run this locally I had to mount /mnt/c to /c:

> sudo mkdir /c
> sudo mount --bind /mnt/c /c

 then change to the /c/ folder in the command line to get the script to run properly

Reply:
I just added volume mounts for the `server/` and `tests/` folders so we can run tests in development without rebuilding the container on every code change.

Good point about docker-compose mount paths in WSL - mounting the /c/ folder has always been necessary to make it work properly.